spec/integration/pg_search_spec.rb

```diff
@@ -139,6 +141,19 @@
         end
       end
     end
+
+    context "when passed a hstore column" do
+      it "builds a scope" do
+        ModelWithPgSearch.pg_search_scope :search_en_description,
+          :against => PgSearch::Configuration::HstoreColumn.new(:description, 'en')
```

Comment:
I'm not a big fan of this syntax. Having to construct `PgSearch::Configuration::HstoreColumn` feels pretty obtuse. That said, I'm glad it works.

Could we maybe do something like detect if the column name is a String containing `->`? Is that a viable option?

Either way, we need some documentation for this feature in the README. I had to dig to figure out how to use it.